internal/workflow_testsuite.go

```diff
@@ -999,10 +1167,10 @@ func (e *TestWorkflowEnvironment) AssertExpectations(t mock.TestingT) bool {
 // It can produce a false result when an argument is a pointer type and the underlying value changed after calling the mocked method.
 func (e *TestWorkflowEnvironment) AssertCalled(t mock.TestingT, methodName string, arguments ...interface{}) bool {
 	dummyT := &testing.T{}
-	if !(e.workflowMock.AssertCalled(dummyT, methodName, arguments...) || e.activityMock.AssertCalled(dummyT, methodName, arguments...)) {
-		return e.workflowMock.AssertCalled(t, methodName, arguments...) && e.activityMock.AssertCalled(t, methodName, arguments...)
-	}
-	return true
```

Comment:
I think this code is wrong, as well as `AssertNotCalled` and `AssertNumberOfCalls`.

Reply:
What do you mean?

Reply:
L1003 checks if both `workflowMock` and `activityMock` was called. I think it was supposed to be if either of them was called. The other functions have similar mistakes.